recipes/sparseassembler/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "SparseAssembler" %}
+{% set version = "20160205" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: "mkdir -p $PREFIX/bin; g++ -O3 -o SparseAssebmler *.cpp; mv SparseAssebmler $PREFIX/bin"
```

Comment:
Looks like typos here, s/ebm/emb/g. That will cause the error you're seeing on travis.